### ISSUE TITLE
Test for repository

### ISSUE THREAD

**Issue by Fobhep** [theforeman/foreman-ansible-modules#193]
The [repository test task ](https://github.com/theforeman/foreman-ansible-modules/blob/master/test/test_playbooks/tasks/repository.yml) first sets the repository_label to a fixed value
```
- repository_label: "just_a_test_repo"
```
However the module call later sets the default behavior to omit. This omit will never be triggered
We should define the variable - if wanted - in the playbook, that includes the  task.

```
label: "{{ repository_label | default(omit) }}"
```

Found @ ForemanHackDay ;)

**Comment by Fobhep** [comment]
We will fix this in #192

**Comment by Fobhep** [comment]
done and merged!
